azkaban-web-server/src/main/resources/conf/azkaban-users.xml

```diff
@@ -1,7 +0,0 @@
-<azkaban-users>
-	<user username="azkaban" password="azkaban" roles="admin" groups="azkaban" />
-	<user username="metrics" password="metrics" roles="metrics"/>
```

Comment:
@fsi206914 
I didn't quite understand your question. Can you please clarify your concern?